global/docker-compose.yml

```diff
@@ -47,16 +47,6 @@ services:
       MAILCATCHER_PORT: 1025
     networks:
       - wplocaldocker
-  phpmyadmin:
-    image: phpmyadmin/phpmyadmin
-    ports:
-      - 127.0.0.1:8092:80
-    depends_on:
-      - mysql
-    environment:
-      PMA_HOST: mysql
-    networks:
-      - wplocaldocker
```

Comment:
@dustinrue, there is the official version of [phpmyadmin](https://hub.docker.com/_/phpmyadmin) that supports ARM platforms. Let's probably not remove this and just update the image to use the official one?